### PR TITLE
AsyncICB need not execute

### PR DESCRIPTION
**Pull request by gregw**
The HttpOutput AsyncICB is a blocking callback, thus it is always executed and
never runs in the selector thread. It thus does not need to execute its call
the Channel.handle

Signed-off-by: Greg Wilkins <gregw@webtide.com>

### CODE REVIEW

**Comment by gregw** [comment]
Hmmm client side failure.  Perhaps it is needed.  Investigating....

**Comment by gregw** [comment]
@sbordet apparently we need to execute from the AsyncICB for HTTP/2 because it is called from the window update handling (it must assume it is non blocking).

**Comment by joakime** [comment]
Does this PR belong to an open Issue?

**Comment by gregw** [comment]
@joakime no it's just a PR.

**Comment by gregw** [comment]
can we merge?

File: jetty-http2/http2-server/src/main/java/org/eclipse/jetty/http2/server/HttpChannelOverHTTP2.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 import org.eclipse.jetty.server.HttpChannel;
 import org.eclipse.jetty.server.HttpConfiguration;
 import org.eclipse.jetty.server.HttpInput;
+import org.eclipse.jetty.server.HttpOutput;
```
Comment (sbordet) [inline jetty-http2/http2-server/src/main/java/org/eclipse/jetty/http2/server/HttpChannelOverHTTP2.java]: I don't think this import is needed ?

**Pull request merged**